content/desktop/extensions/private-marketplace.md

```diff
@@ -0,0 +1,179 @@
+---
+description: Extensions
+keywords: Docker Extensions, Docker Desktop, Linux, Mac, Windows,
+title: Configure an extensions private marketplace
+---
+
+This page shows administrators how to configure a private extensions marketplace with a curated list of extensions for your Docker Desktop users.
+
+Extensions private marketplace is designed specifically for organizations who don’t give developers root access to their machines.
```

Comment:
I don't think we need to mention this. If user are root, the private marketplace will stil work and provide the curated list of extensinos, but user will have ways to bypass it and install extensions outside of the marketplace. If admins are ready to give root access to users, then that should not be an issue